Yoast/wordpress-seo
# Checks if schema attributes exist

## Because of some problems, it was easier to create a new PR. [The new PR can be found here](https://github.com/Yoast/wordpress-seo/pull/13789).

## Summary

Added `isset` checks for values which can be non existing, when they're left empty.


This PR can be summarized in the following changelog entry:

* Fixes a bug where a notice would be thrown when fields in the FAQ or HowTo blocks were left empty.

## Relevant technical choices:

* Added checks to assure all required keys exist. If not, `return`.

## Test instructions

This PR can be tested by following these steps:

* Create a new post with an empty Faq block --> Publish.
    * No notices or warnings should be thrown.
    * Fill the Faq block with data, it should output this data and no warnings or notices should be thrown.

* Create a new post with an empty HowTo block --> Publish.
    * No notices or warnings should be thrown.
    * Fill the HowTo block with data, it should output this data and no warnings or notices should be thrown.


## UI changes

## Documentation

## Quality assurance


Fixes #
